pkg/lib/cockpit.d.ts

```diff
@@ -29,6 +29,8 @@ declare module 'cockpit' {
         toString(): string;
     }
 
+    type Superuser = "require" | "try" | null | undefined;
```

Comment:
I was going to say: This is a very generic name, can we find something more specific? But it's not exported so I guess it doesn't matter much.

Reply:
I like `SuperuserMode`, or `SuperuserType`.  I had `SuperuserAccessLevel`, but `Mode` is much better!

Reply:
Mode it is!